### PR TITLE
Add Teshari from Polaris

### PR DESCRIPTION
**Pull request by tigercat2000**
These cute little critters are a new publically available species.
They're raptorjesus incarnate! More birds never hurt anyone, right? Their sprite is
shorter than humans, requiring more sprite_sheets, most of which I've
stolen from Polaris.

Features of the race:
 - Can hide under tables
 - Go faster than humans (mildly)
 - Brute & Burn weakness of 133%
 - They cannot push large mobs (To be implemented, right now they just
 can't push any human mob that isn't a "small" species)
 - They have a verb to "listen to their surroundings", getting an
 indication of mob movement going on in a radius around them. This isn't
 nearly as powerful as thermal vision, as all you can tell is the
 general distance between you and the mob, and a cardinal direction
 towards it.
 - They're fucking adorable
 - More SKREEING FOR EVERYONE
 - Omnivores, currently, can be changed
 - Health is half of a regular human, 50 max
 - They can walk under tables, and hide under them with a verb in the IC
 tab (remember old nymphs?)
 - They speak Schechi, another chirp-based language
 - Birdies have claws to fight off the kitties

### CODE REVIEW

**Comment by Rumiluntti** [comment]
>Can hide under tables
>Go faster than humans (mildly)

Haha, fuck no.

>They're fucking adorable
According to the pictures they look liek sentient genital warts

**Comment by SamHPurp** [comment]
PR #7777 - meme PR

**Comment by Rumiluntti** [comment]
You can never be too sure.

**Comment by scrubmcnoob** [comment]
can you code in a feature where you can grab intent them into a near choke hold and drop kick them across the station?
i need it for muh immersions. @tigercat2000 

just in case this isn't a meme pr and /real/ feedback is needed

 they definitely mix up the available species as they are nothing like a human or any other species for that matter. They are like the opposite of Diona which I like. Diona being more tanky species and these being more mobile and fragile.
I'm not so sure about hiding under tables, feel like this be too easy for abuse. Just hide under a table with a baton out and you can whack any pursuer before they have the chance to alt click your tile and then click your sprite.

**Comment by granodd** [comment]
Part of what makes emagged drones really powerful is their ability to hide under tables. Given it sort of makes sense, as they don't have a lot of health, and neither do Teshari, but with the right tool it can be very powerful.

**Comment by KingPhilipIII** [comment]
Like  @scrubmcnoob says, I like the fact that it actually mixes things up by introducing a species that is distinctly unique with it's mechanics and stats. 

It also means players have to learn to apply new tactics for handling people who use their unique abilities aggressively or to excess.

**Comment by Jaywings** [comment]
Sure.

**Comment by LightFire53** [comment]
These sprites are... not good, in my opinion...

@tigercat2000 do you have the other three angles?

Still think resomi would be a better option than these...

**Comment by R1f72r** [comment]
I really like it, it adds more unique stuff to the station, which is what i like.

**Comment by ProperPants** [comment]
2017-07-12 22_57_18-adds in kidan variants wip by shazbot194 pull request 7665 paradisess13_p

**Comment by MarcellusPye** [comment]
Should I even explain Meme PR?

**Comment by taukausanake** [comment]
OH. SO YOU WON'T PORT RESOMI BUT WHATEVER THE FACK THESE THINGS ARE IS FINE?

Man, I wanted to steal this PR number. Didn't have anything at the time though. Oh well.
Yeah, even if serious I wouldn't port these before Resomi. Those things are WAY more cuter than these

**Comment by tigercat2000** [comment]
@taukausanake 
>Resomi are way more cuter

Same sprites....

**Comment by taukausanake** [comment]
@tigercat2000 They don't look it. Missing the arm feathers and pretty colors. For all I know these are Chinese knockoff Resomi which I'm betting they are now that you pointed the sprites out

**Comment by Bigtimetripod** [comment]
KILL IT WITH FIRE BEFORE IT LAYS EGGS!

**Comment by matt81093** [comment]
Can they lay eggs for the chef to cook up?

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -108,6 +108,8 @@
 
 	var/obj/item/weapon/card/id/guest/guest_pass = null // Guest pass attached to the ID
 
+	sprite_sheets = list("Teshari" = 'icons/mob/species/seromi/id.dmi')
```
Comment (outbools) [inline code/game/objects/items/weapons/cards_ids.dm]: >seromi
totally not a resomi bootleg

Comment (taukausanake) [inline code/game/objects/items/weapons/cards_ids.dm]: Totally blaming Space China now. This is why we can't have nice things

**Comment by KasparoVy** [comment]
Something I'd shill 200KP for.

**Comment by KasparoVy** [comment]
Of note, "weak stomp" was also omitted.

@tigercat2000 Refer [here](https://github.com/PolarisSS13/Polaris/search?utf8=%E2%9C%93&q=body_hair&type=) for pointers on how to implement the "feathers" (if that's even something that has a chance of being undertaken here). Not much involved at all.

File: code/game/objects/items/weapons/cards_ids.dm (hunk 1 above)
Comment (KasparoVy) [inline code/game/objects/items/weapons/cards_ids.dm]: > Refer [**_here_**](https://github.com/PolarisSS13/Polaris/search?utf8=%E2%9C%93&q=body_hair&type=) (Polaris body_hair code, not much involved) for pointers on how to implement the "feathers" (if that's even something that has a chance of being undertaken here). Not much involved at all.

**Comment by Fox-McCloud** [comment]
THE MEMES WILL NEVER DIE!

**Comment by Fox-McCloud** [comment]
actually, they just did...

🤔

**Pull request closed without merging**